drivers/video/fbdev/bcm2708_fb.c

```diff
@@ -82,62 +85,130 @@ struct bcm2708_fb_stats {
 	u32 dma_irqs;
 };
 
+struct vc4_display_settings_t {
+	u32 display_num;
+	u32 width;
+	u32 height;
+	u32 pitch;
+	u32 depth;
+	u32 virtual_width;
+	u32 virtual_height;
+	u32 virtual_width_offset;
+	u32 virtual_height_offset;
+	unsigned long fb_bus_address;
+};
+
+/* One static instance of the DMA parameters, since they are used by all
+ * bcm FB's
+ */
+static int dma_chan;
```

Comment:
Do these really have to be static? Allocate a parent object and populate that in probe.